cram/cram_encode.c

```diff
@@ -1923,8 +1930,13 @@ int cram_encode_container(cram_fd *fd, cram_container *c) {
     }
 
     c->ref_seq_id    = c->slices[0]->hdr->ref_seq_id;
-    c->ref_seq_start = c->slices[0]->hdr->ref_seq_start;
-    c->ref_seq_span  = c->slices[0]->hdr->ref_seq_span;
+    if (c->ref_seq_id == -1 && CRAM_ge31(fd->version)) {
```

Comment:
It might be a good idea to have a short comment here, explaining why there's a check on the file version (and also in `cram_update_current_slice`).

Reply:
Could do, but it'd probably be best to comment it to simply see the relevant commit message as it'd be unwieldy to document the full reasoning.  I'll try and think of something appropriate.